sdk/search/Azure.Search.Documents/src/Generated/Models/SearchDocumentsResult.Serialization.cs

```diff
@@ -141,16 +140,6 @@ internal static SearchDocumentsResult DeserializeSearchDocumentsResult(JsonEleme
                     searchSemanticQueryRewritesResultType = new SemanticQueryRewritesResultType(property.Value.GetString());
                     continue;
                 }
-                if (property.NameEquals("@search.debug"u8))
```

Comment:
This property wasn't removed, another question about what the generator is doing.